unlock-app/src/components/creator/lock/LockIconBar.tsx

```diff
@@ -1,21 +1,31 @@
 import React, { useContext } from 'react'
-import styled from 'styled-components'
 import PropTypes from 'prop-types'
 
 import Buttons from '../../interface/buttons/lock'
 import UnlockPropTypes from '../../../propTypes'
 import CreatorLockStatus from './CreatorLockStatus'
-import Media from '../../../theme/media'
 import withConfig, { ConfigContext } from '../../../utils/withConfig'
-import { TransactionType, TransactionStatus } from '../../../unlockTypes'
 import { AuthenticationContext } from '../../../contexts/AuthenticationContext'
 
-export function LockIconBar({ lock, toggleCode, withdraw, toggleCreditCard }) {
+interface LockIconBarProps {
+  lock: any
+  withdraw: any
```

Comment:
Type the withdraw (it should be a function based on the Button.Withdraw component) and lock (from unlockTypes or type package?).